packages/react-components/package.json

```diff
@@ -48,7 +48,7 @@
     "@fluentui/react-input": "9.0.0-beta.0",
     "@fluentui/react-label": "9.0.0-beta.4",
     "@fluentui/react-link": "9.0.0-beta.5",
-    "@fluentui/react-make-styles": "9.0.0-beta.4",
+    "@griffel/react": "1.0.0",
```

Comment:
Worth a separate discussion but I would expect a caret version here as griffel is now external dependency following semver.

Reply:
I decided to keep it as it is for now. But yeah, this should be discussed 👍